cloud/amazon/iam_policy.py

```diff
@@ -283,7 +283,7 @@ def main():
       iam_name=dict(default=None, required=False),
       policy_name=dict(default=None, required=True),
       policy_document=dict(default=None, required=False),
-      policy_json=dict(default=None, required=False),
+      policy_json=dict(type='dict', default=None, required=False),
```

Comment:
@bcoca I gave an example here. https://github.com/ansible/ansible-modules-core/pull/4298#issuecomment-278773421.  

There error response is:
```
fatal: [localhost]: FAILED! => {
    "changed": true, 
    "failed": true, 
    "invocation": {
        "module_args": {
            "aws_access_key": null, 
            "aws_secret_key": null, 
            "ec2_url": null, 
            "iam_name": "kubernetes-iam-role-qa", 
            "iam_type": "role", 
            "policy_document": null, 
            "policy_json": "{'Version': '2012-10-17', 'Statement': [{'Action': ['swf:*'], 'Resource': ['arn:aws:swf:*:1234123412341:/domain/qa'], 'Effect': 'Allow', 'Sid': 'Stmt1486673523000'}]}", 
            "policy_name": "qaSwfAllAccessWithinDomain", 
            "profile": null, 
            "region": null, 
            "security_token": null, 
            "skip_duplicates": true, 
            "state": "present", 
            "validate_certs": true
        }, 
        "module_name": "iam_policy"
    }, 
    "msg": "Syntax errors in policy."
}
```